README.md

```diff
@@ -6,7 +6,6 @@
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
   - [Prerequisites](#prerequisites)
```

Comment:
![image](https://github.com/katfogy/vet-clinic/assets/93039359/32ecad55-6b8e-4d01-bd4c-1b7d8bc5ad2f)

- [ ] As it is suggested previously, It would be nice if you fix all the links highlighted above to make your readme more professional and descriptive 👍 because right now these highlighted links are broken 😨 Kindly fix them or remove them if you don't have any dedication sections for these links 🤔